src/test/java/pl/techlab24/OSKManager/model/validation/RideDetailsValidatorTest.java

```diff
@@ -60,6 +77,21 @@ void shouldValidateNullRideType() {
         assertEquals(Collections.singletonList("Ride type cannot be null."), resultOfValidation);
     }
 
+    @Test
+    void shouldValidateCourseClientMethodCallValidateMethodFromCourseClientValidatorClass() {
+        try (MockedStatic<CourseClientValidator> mockedStatic = mockStatic(CourseClientValidator.class)) {
+            // given
+            mockedStatic.when(() -> CourseClientValidator.validate(correctCourseClient)).thenReturn(Collections.emptyList());
+
+            // when
+            List<String> result = RideDetailsValidator.validate(correctRideDetails);
+
+            // then
+            assertEquals(Collections.emptyList(), result);
+            mockedStatic.verify(() -> CourseClientValidator.validate(correctCourseClient));
+        }
+    }
```

Comment:
Shouldn't we stick to the rule that we try to put things related to fields in the same order as those fields appear? If so, this test should be the last one, because it's related to CourseClient, and CourseClient is the last field in RideDetails class. What do you think?

Reply:
ok I will move it